backend/readers_backend/readers_backend/templates/index.html

```diff
@@ -5,17 +5,218 @@
+        background-color: #ffffff;
+        padding: 1.25rem 2rem; /* 20px 32px */
+        box-shadow: 0 1px 3px rgba(0, 0, 0, 0.05), 0 1px 2px rgba(0, 0, 0, 0.03);
+        text-align: center;
+        border-bottom: 1px solid #e5e7eb; /* Subtle border */
+    }
+
+    .logo {
+        max-height: 48px; /* Slightly reduced for a sleeker look */
+        width: auto;
+        vertical-align: middle;
+    }
+
+    /* Main Content Container */
+    .container {
+        max-width: 1280px; /* Common max-width for modern layouts */
+        margin: 0 auto;
+        padding: 2.5rem 2rem; /* 40px 32px */
+        flex-grow: 1; /* Allows container to expand and push footer down */
+        width: 100%;
+    }
+
+    /* Workbook Grid */
+    .workbook-container {
+        display: grid;
+        grid-template-columns: repeat(auto-fill, minmax(240px, 1fr)); /* Slightly larger cards */
+        gap: 2rem; /* 32px */
+    }
+
+    /* Individual Workbook Link/Card */
+    .workbook-link {
+        display: block;
+        position: relative;
+        border-radius: 12px; /* Softer, more modern rounding */
+        overflow: hidden;
+        background-color: #ffffff; /* Card background */
+        box-shadow: 0 4px 6px -1px rgba(0, 0, 0, 0.07), 0 2px 4px -2px rgba(0, 0, 0, 0.04);
+        transition: transform 0.25s ease-in-out, box-shadow 0.25s ease-in-out;
+        text-decoration: none; /* Remove underline from link */
+    }
+
+    .workbook-link:hover,
+    .workbook-link:focus-visible { /* Added focus-visible for accessibility */
+        transform: translateY(-6px);
+        box-shadow: 0 10px 15px -3px rgba(0, 0, 0, 0.08), 0 4px 6px -4px rgba(0, 0, 0, 0.05);
+    }
+
+    .workbook-cover {
+        width: 100%;
+        height: auto; /* Maintain aspect ratio */
+        aspect-ratio: 3 / 4; /* Common book cover aspect ratio, adjust if needed */
+        object-fit: cover; /* Ensures image covers the area without distortion */
+        display: block;
+        background-color: #e5e7eb; /* Placeholder color for images */
+    }
+
+    /* Workbook Overlay */
+    .workbook-overlay {
+        position: absolute;
+        inset: 0; /* top, right, bottom, left = 0 */
+        background: linear-gradient(to top, rgba(17, 24, 39, 0.85) 0%, rgba(17, 24, 39, 0.3) 60%, transparent 100%); /* Dark gradient overlay */
+        display: flex;
+        flex-direction: column;
+        justify-content: flex-end; /* Align text to the bottom */
+        align-items: center; /* Center text horizontally */
+        opacity: 0;
+        transition: opacity 0.25s ease-in-out;
+        padding: 1.25rem 1rem; /* Padding for the text inside */
+        text-align: center;
+    }
+
+    .workbook-link:hover .workbook-overlay,
+    .workbook-link:focus-visible .workbook-overlay {
+        opacity: 1;
+    }
+
+    .workbook-overlay span {
+        color: #ffffff;
+        font-weight: 500;
+        font-size: 1rem; /* Slightly larger for better readability on overlay */
```

Comment:
How is this larger? I didn’t see anything reducing it from 1rem anywhere else.

Reply:
Ah, it comes after this comment. Not the most obvious order for top to bottom reading.